test/stat_tracker_test.rb

```diff
@@ -0,0 +1 @@
+require './test/test_helper'
```

Comment:
Thank you for adding the test helper to our stattracker.  Should we add more files to the Test Helper file so we keep the top of our tests clean?